searchlib/src/vespa/searchlib/queryeval/nearest_neighbor_iterator.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "nearest_neighbor_iterator.h"
+
+using vespalib::ConstArrayRef;
+using vespalib::tensor::TypedCells;
+
+namespace {
+
+struct SumSq
```

Comment:
Consider using full name to make code clearer: SumSquared

Reply:
done.